core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java

```diff
@@ -277,7 +277,16 @@ public static String loadGenePanelInformation(File file) throws Exception {
         return properties.getProperty("gene_panel");
     }
 
-	public static String loadFilteredMutations(File file) throws Exception {
+	/**
+	 * Gets the information of "variant_classification"filter" in the file, if it exists. Otherwise, it
+	 * returns null. "variant_classification_filter" can be used in the mutation meta file to specify
+	 * which types of mutations want to be filtered.
+	 * 
+	 * @param file
+	 * @return a string with the types of mutations that want to be filtered, comma-separated.
```

Comment:
maybe rephrase "types of mutations that want to be filtered" -> types of mutations that should be filtered